packages/react/src/components/Storage/FileUploader/Tracker/index.tsx

```diff
@@ -54,106 +54,86 @@ export function Tracker({
+            <Text className={ComponentClassNames.FileUploaderFileName}>
+              {name}
+            </Text>
+            <FileState
+              error={isError}
+              errorMessage={errorMessage}
+              success={isSuccess && !isError}
+              paused={isPaused}
+              loading={isLoading}
+              percentage={percentage}
+            />
+          </View>
+
+          {showonStartEdit() && (
+            <Button onClick={onStartEdit} size="small" variation="link">
+              <IconEdit fontSize="medium" />
             </Button>
-          </Flex>
-        ) : (
-          <>
-            <Flex
-              direction="column"
-              className="amplify-fileuploader__file-content"
-              flex="1"
-              gap="0"
-            >
-              <Flex
-                direction="row"
-                className="amplify-fileuploader__file-content"
-                gap="xxs"
-              >
-                <Text
-                  as="span"
-                  fontWeight="bold"
-                  className="amplify-fileuploader__filename"
-                >
-                  {name}
-                </Text>
+          )}
 
-                {showonStartEdit() && (
-                  <Button onClick={onStartEdit} size="small" variation="link">
-                    <EditIcon fontSize="medium" />
-                  </Button>
-                )}
-                <Text as="span" color="font.tertiary" marginInlineStart="small">
-                  {humanFileSize(size, true)}
-                </Text>
-              </Flex>
-              <FileState
-                error={isError}
-                errorMessage={errorMessage}
-                success={isSuccess && !isError}
-                paused={isPaused}
-                loading={isLoading}
-              />
-            </Flex>
-            {isLoading && (
-              <>
-                {isPaused ? (
-                  <Button onClick={onResume} size="small" variation="link">
-                    {translate('Resume')}
-                  </Button>
-                ) : (
-                  <Button onClick={onPause} size="small" variation="link">
-                    {translate('pause')}
-                  </Button>
-                )}
-              </>
-            )}
-            {/* {isSuccess && !isError && (
+          <Text as="span" className={ComponentClassNames.FileUploaderFileSize}>
+            {humanFileSize(size, true)}
+          </Text>
+          {isLoading && (
+            <>
+              {isPaused ? (
+                <Button onClick={onResume} size="small" variation="link">
+                  {translate('Resume')}
+                </Button>
+              ) : (
+                <Button onClick={onPause} size="small" variation="link">
+                  {translate('pause')}
+                </Button>
+              )}
+            </>
+          )}
+          {/* {isSuccess && !isError && (
```

Comment:
What's the comment for?

Reply:
I think because I moved things around Git thinks I added this, but this comment was there previously: https://github.com/aws-amplify/amplify-ui/blob/fileuploader/main/packages/react/src/components/Storage/FileUploader/Tracker/index.tsx#L133 I'll ask @ErikCH about it

Reply:
Yes, this has been deleted in the refactor #2892. The refactor will go in first, and then @dbanksdesign will need to merge back in the changes from `file-uploader/main` back into this branch, and this will be deleted.  The refactor will also fix the e2e tests